packages/core/src/textures/Texture.js

```diff
@@ -102,13 +102,21 @@ export default class Texture extends EventEmitter
         this.requiresUpdate = false;
 
         /**
-         * The WebGL UV data cache.
+         * The WebGL UV data cache. Can be used as quad UV
          *
          * @member {PIXI.TextureUvs}
          * @private
          */
         this._uvs = null;
 
+        /**
+         * Default TextureMatrix instance for this texture
+         * By default that object is not created because its heavy
+         *
+         * @member {PIXI.TextureMatrix}
+         */
+        this.uvMatrix = null;
```

Comment:
What about renaming this to `textureTransform` instead of `uvMatrix`?

Reply:
Because its not Transform, it cant be combined with other Transforms, its just single Matrix. I want it to be as verbose as possible.

Reply:
Flash has Transform and ColorTransform, and both have parent and composition rules. This thing cant be composed with anything, its just a Matrix